Source/WebCore/rendering/GlyphDisplayListCache.cpp

```diff
@@ -170,7 +171,8 @@ DisplayList::DisplayList* GlyphDisplayListCache::getIfExists(const InlineDisplay
 
 void GlyphDisplayListCache::remove(const void* run)
 {
-    m_entriesForLayoutRun.remove(run);
+    RefPtr value = m_entriesForLayoutRun.take(run);
+    m_entriesToAsyncDelete.append(WTFMove(value));
```

Comment:
Does this increase the memory working set?  Can this increase the possibility of jetsam on iOS?

Reply:
Should we try to limit the number of entries of `m_entriesToAsyncDelete`?
```
if (m_entriesToAsyncDelete.size() > SOME_MAX)
    m_entriesForLayoutRun.remove(run);
else
    m_entriesToAsyncDelete.append(m_entriesForLayoutRun.take(run));
```